python/dlr/dlr_model.py

```diff
@@ -120,15 +120,19 @@ def __init__(self, model_path, dev_type='cpu', dev_id=0):
         self.num_inputs = self._get_num_inputs()
         self.num_weights = self._get_num_weights()
         self.input_names = []
+        self.output_names = []
         self.weight_names = []
         self.input_shapes = {}   # Remember shape used in _set_input()
-        for i in range(self.num_inputs):
-            self.input_names.append(self._get_input_name(i))
+        self.output_dtypes = []
```

Comment:
Lets add `input_dtypes` as well. `GetDLRInputType` was added to C API recently.

Reply:
👍